### ISSUE TITLE
Question about the reproducing result on vtab-1k using ViT-B/16 pre-trained on ImageNet-21k.

### ISSUE THREAD

**Issue by wildboarman** [ziplab/SPT#1]
Hi, great work! We are unable to achieve the results of the experiment described in the title of the paper. But we can reproduce the results of pre-training the model using other datasets. Can you give us some suggestions for experiments? The following are the experimental results under the parameter quantity of 0.4M. Looking forward to hearing from you.

70.82, 92.42, 71.54, 99.28, 87.22, 55.20, 91.22,Natural.
85.57, 95.96, 85.60, 74.31,Specialized.
81.83, 66.81, 49.36, 78.76, 79.02, 49.38, 27.73, 38.11,Structured.

**Comment by Charleshhy** [comment]
Hi,

Thanks for your interest in our work! Empirically we find that using more trainable parameters does not always get us the best results. Considering this, we sweep through trainable parameters 0.2M, 0.4M, 0.6M, and 0.8M on the validation set and report the result of the best-performing one.

Regards,
Haoyu

**Comment by wildboarman** [comment]
Thank you for your help! 
So I can understand that the parameter constraints used in different datasets in the paper results may not necessarily be the same, for example, Cifar may have used 0.2M and DTD may have used 0.4M. Is this understanding correct?

**Comment by Charleshhy** [comment]
> Thank you for your help! So I can understand that the parameter constraints used in different datasets in the paper results may not necessarily be the same, for example, Cifar may have used 0.2M and DTD may have used 0.4M. Is this understanding correct?

Yes, you are correct. You can see the number of trainable parameters as a hyper-parameter.

**Comment by Charleshhy** [comment]
Close this issue due to no further activity.
